Tools/DeltaV/tidy_map.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python3
+"""
+Space Station 14 Map Tidying Tool
+Original work Copyright (c) 2023 Magil
+Modified work Copyright (c) 2024 DeltaV-Station
+
+This script is licensed under MIT
+Modifications include code modernization, restructuring, and YAML handling updates
+"""
+
+import argparse
+import locale
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Any
+from ruamel.yaml import YAML
+
+
+# Configuration for components that should be handled during tidying
+class TidyConfig:
+    # Components that should be removed entirely
+    REMOVE_COMPONENTS: List[str] = [
+        "AmbientSound",
+        "EmitSoundOnCollide",
+        "Fixtures",
+        "GravityShake",
+        "HandheldLight",  # Floodlights are serializing these?
+        "PlaySoundBehaviour",
+    ]
+
+    # Components that will have specific fields removed and may be removed entirely if empty
+    REMOVE_COMPONENT_DATA: Dict[str, List[str]] = {
+        "Airtight": ["airBlocked"],
```

Comment:
these components should be checked for which are TimeOffsetSerializer and can be removed